File: UserProject/src/main/java/com/botthebuilder/userproject/entities/Action.java
package com.botthebuilder.userproject.entities;

import lombok.*;

import javax.persistence.*;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@ToString(exclude = "project")
public class Action {

    @Id
    @SequenceGenerator(
            name = "action_sequence",
            sequenceName = "action_sequence"
    )
    @GeneratedValue(
            strategy = GenerationType.AUTO,
            generator = "action_sequence"
    )
    private Long actionId;

    @ManyToOne(
            optional = false
    )
    @JoinColumn(
            name = "projectId",
            referencedColumnName = "projectId"
    )
    private Project project;

    private String actionName;

    private String actionDetail;

}

File: PythonMainServer/models/Action.py
from db import db 

class Action(db.Model):
	action_id = db.Column(db.BigInteger, primary_key = True)
	project_id = db.Column(db.BigInteger, db.ForeignKey('project.project_id'), nullable = False)
	action_name = db.Column(db.String(20), nullable = True)
	action_detail = db.Column(db.String(80), nullable = True)